RentallCarsAPI/Controllers/ClientController.cs

```diff
@@ -92,6 +92,96 @@ public IActionResult GetAll()
             }
             response.Succes = true;
             response.Message = "Found successfully";
+            response.Data = clients;
+            return Ok(response);
+        }
+
+        [HttpPut]
+        public IActionResult Update(ClientRequest model)
```

Comment:
Acá podríamos utilizar un nombre mas representativo
```suggestion
        public IActionResult Update(ClientRequest clientRequest)
```